doc/util/litchapel.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+from __future__ import print_function
+
+import sys
+
+def titlecomment(line):
```

Comment:
`title_comment` to fit python convention

Reply:
Can do. Since I was simply splitting out the file, I wanted to minimize extraneous changes (such as naming).